contracts/interfaces/0.8.x/IMintingAPIMinterV0.sol

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+// Created By: Art Blocks Inc.
+
+pragma solidity ^0.8.0;
+
+interface IEngineContractMintingAPIV0 {
```

Comment:
Could we add a comment block above the interface definition here that clarifies why this interface is needed (e.g. for compatibility verification with the minting API assumptions)?